packages/core/android/src/test/kotlin/com/datadog/tools/unit/MapExt.kt

```diff
@@ -20,3 +22,13 @@ fun Map<*, *>.toReadableMap(): ReadableMap {
     // this FB implementation is not backed by Android-specific .so library, so ok for unit tests
     return JavaOnlyMap.of(*keysAndValues.toTypedArray())
 }
+
+fun ArrayList<*>.toReadableArray(): ReadableArray {
```

Comment:
in theory this one is not needed, because `ArrayList` is child of `List` and there is a a `List<*>.toReadableArray()` just below